### PR TITLE
Add CMake

### PR DESCRIPTION
**Pull request by Ingvord**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 2.8.9)
+project(DataBase)
+
+include(configure/CMakeLists.txt)
+
+set(MAJOR_VERSION "5")
+set(MINOR_VERSION "3")
```
Comment (bourtemb) [inline CMakeLists.txt]: The current version should actually be 5.6.0 as on Sourceforge.
This change was forgotten in the Makefile (which should be updated too) after the preparation of Tango 9.2.5 release which was done from Sourceforge.

**Review by bourtemb — COMMENTED** [review]
It looks like the compilation options and linker options which were used before are not taken into account when compiling with CMake.
Were they simply forgotten or was it done on purpose?

**Comment by Ingvord** [comment]
This PR is almost 1-year old... I actually do not remember, but most probably it was forgotten.

**Comment by bourtemb** [comment]
Looks good to me as a good starting point.

There are still some missing features like the generation of the database creation scripts. This could be eventually done in a separate PR.

There will be some work to do to support other databases like maria-db (using libmariadb-client-lgpl-dev package on Debian) but this can be done in separate PRs too.

**Pull request merged**